### PR TITLE
Org profile readme.

### PR DESCRIPTION
**Pull request by muzam1l**
[Preview](https://github.com/runziggurat/.github/blob/ceb389e07eec2148d4cb7cd3b39c425d42bf7851/profile/README.md)

### CODE REVIEW

**Comment by muzam1l** [comment]
@aphelionz Is this the template we are trying to go for, or we need something different?

**Comment by aphelionz** [comment]
That's what we're going for! Keep up the good work. Perhaps call it the "Coverage Table" or some such thing.

**Comment by ljedrz** [comment]
I'd reorder the columns a little bit, so that the conformance checks are first.

We might also want to add a "started?" column or sth like that.

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,28 @@
-# Ziggurat
+<p align="center">
+    <img src="assets/logo.png" height="300px">
+</p>
+<br />
+Ziggurat is a collection of network testing tools and a suite of conformance, performance and resistance tests our team developed in Rust.
+
+# About
+
+The project is developed by [Equilibrium](https://equilibrium.co) that provides [zcashd](https://github.com/zcash/zcash) and [zebra](https://github.com/ZcashFoundation/zebra) developers with the foundation of reliability, performance, and scalability needed to realize the true potential of the Zcash network.
```
Comment (niklaslong) [inline profile/README.md]: We should make this generic—we're not only testing Zcash these days 😉

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (niklaslong) [inline .vscode/settings.json]: Don't think this file should be included?

Comment (aphelionz) [reply .vscode/settings.json]: you can add this to your local global `~/.gitignore` file

File: profile/README.md (hunk 1 above)
Comment (aphelionz) [reply profile/README.md]: The full list is ZCash, Algorand, and XRPL Ledger. Hopefully Polkadot soon too 🤞

File: profile/README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+|--------------------|--------|-------------|----------|-------------|-----------------|--------|
+| ZCash              | Yes    | No          | Kinda    | No          | Yes (de-facto)  | Yes    |
+| Algorand           | No     | No          | No       | No          | No              | No     |
+| Aleo               | No     | In Progress | No       | In Progress | No              | No     |  
+| Polkadot           | No     | No          | No       | No          | No              | No     |
+| Cardano            | No     | No          | No       | No          | No              | No     |
+| Solana             | No     | No          | No       | No          | No              | No     |
+| Mina               | No     | No          | No       | No          | No              | No     |
+| Eth2 Beacon Chain  | No     | No          | No       | No          | No              | No     |
+| Pathfinder         | No     | No          | No       | No          | No              | No     |
```
Comment (aphelionz) [inline profile/README.md]: Let's remove Cardano, Pathfinder, and Mina from the list _for now_

File: profile/README.md [hunk 4]
```diff
@@ -1 +1,28 @@
...
+
+
+# Contributing [*Or something else*]
+
+[TODO]
```
Comment (aphelionz) [inline profile/README.md]: For now just say that issues and PRs are accepted

**Comment by muzam1l** [comment]
Pardon this review adding removing, GitHub things!!

**Comment by ljedrz** [comment]
shouldn't we add XRPL to the table too, right after Zcash?

**Comment by muzam1l** [comment]
> shouldn't we add XRPL to the table too, right after Zcash?

Ok, and which columns to to tick or say started or something?

**Comment by ljedrz** [comment]
> which columns to to tick or say started or something?

it's in progress; we do have a few conformance tests already, but they're not comprehensive yet

**Comment by muzam1l** [comment]
> > which columns to to tick or say started or something?
> 
> it's in progress; we do have a few conformance tests already, but they're not comprehensive yet

So for XRPL, we should have "In progress" for **Spec Conformance** and "No" in rest of the columns, right?

**Comment by aphelionz** [comment]
How's this looking @niklaslong @ljedrz? Can we merge before the weekend?

**Comment by ljedrz** [comment]
One last thing: in Aleo, while we did have a crawler, we will need to have it rewritten for testnet3. Not sure if this amounts to "in progress".

**Comment by ljedrz** [comment]
other than that, LGTM

**Comment by muzam1l** [comment]
> ...while we did have a crawler, we will need to have it rewritten for testnet3. Not sure if this amounts to "in progress"

Kinda does 🤔

**Comment by ljedrz** [comment]
@aphelionz your call - if having an impl that can't currently be used (as the P2P connections are not re-enabled in the testnet yet)  and will need to be adjusted to the changed network protocol is good enough for "in progress", I'm ok with this :smiley:.

File: profile/README.md [hunk 5]
```diff
@@ -1 +1,26 @@
...
+    <img src="assets/logo.png" height="300px">
+</p>
+<br />
+Ziggurat is a collection of network testing tools and a suite of conformance, performance and resistance tests our team developed in Rust.
+
+# About
+
+The project is developed by [Equilibrium](https://equilibrium.co) that provides the target networks with the foundation of reliability, performance, and scalability needed to realize the true potential of the network. Our clients  include, but not limited to, ZCash, Algorand, and XRPL Ledger.
+
+# Covergage
```
Comment (niklaslong) [inline profile/README.md]:
```suggestion
# Coverage
```

**Comment by niklaslong** [comment]
Do we want to include chains we don't yet have contracts with (e.g. Solana)?

File: profile/README.md [hunk 6]
```diff
@@ -1 +1,26 @@
...
+# Coverage
+
+|       | **Spec Conformance** | **Fuzzing** | **Crawler** | **Chaos + Malicious Actor Modeling** | **Visualization** | **Performance Benchmarking (loosely defined)** |
+|--------------------|--------|-------------|----------|-------------|-----------------|--------|
+| ZCash              | Yes (de-facto)  | Yes    | No          | Kinda    | No          | Yes    |
+| XRPL Ledger        | In progress     | No     | No          | No       | No          | No     |
+| Algorand           | No              | No     | No          | No       | No          | No     |
+| Aleo               | No              | No     | In Progress | No       | In Progress | No     |  
+| Polkadot           | No              | No     | No          | No       | No          | No     |
+| Solana             | No              | No     | No          | No       | No          | No     |
```
Comment (aphelionz) [inline profile/README.md]:
```suggestion
```

File: profile/README.md [hunk 7]
```diff
@@ -1 +1,26 @@
...
+| Eth2 Beacon Chain  | No              | No     | No          | No       | No          | No     |
```
Comment (aphelionz) [inline profile/README.md]:
```suggestion
```

File: profile/README.md [hunk 8]
```diff
@@ -1 +1,26 @@
...
+
+# About
+
+The project is developed by [Equilibrium](https://equilibrium.co) that provides the target networks with the foundation of reliability, performance, and scalability needed to realize the true potential of the network. Our clients  include, but not limited to, ZCash, Algorand, and XRPL Ledger.
+
+# Coverage
+
+|       | **Spec Conformance** | **Fuzzing** | **Crawler** | **Chaos + Malicious Actor Modeling** | **Visualization** | **Performance Benchmarking (loosely defined)** |
+|--------------------|--------|-------------|----------|-------------|-----------------|--------|
+| ZCash              | Yes (de-facto)  | Yes    | No          | Kinda    | No          | Yes    |
```
Comment (aphelionz) [inline profile/README.md]:
```suggestion
| ZCash              | Yes  | Yes    | In Progress | Yes          | In Progress          | Yes    |
```

File: profile/README.md [hunk 9]
```diff
@@ -1 +1,26 @@
...
+<br />
+Ziggurat is a collection of network testing tools and a suite of conformance, performance and resistance tests our team developed in Rust.
+
+# About
+
+The project is developed by [Equilibrium](https://equilibrium.co) that provides the target networks with the foundation of reliability, performance, and scalability needed to realize the true potential of the network. Our clients  include, but not limited to, ZCash, Algorand, and XRPL Ledger.
+
+# Coverage
+
+|       | **Spec Conformance** | **Fuzzing** | **Crawler** | **Chaos + Malicious Actor Modeling** | **Visualization** | **Performance Benchmarking (loosely defined)** |
```
Comment (aphelionz) [inline profile/README.md]:
```suggestion
|       | **Spec Conformance** | **Fuzzing** | **Crawler** | ** Malicious Actor Modeling** | **Visualization** | **Performance Benchmarking ** |
```

File: profile/README.md [hunk 10]
```diff
@@ -1 +1,26 @@
...
+|--------------------|--------|-------------|----------|-------------|-----------------|--------|
+| ZCash              | Yes (de-facto)  | Yes    | No          | Kinda    | No          | Yes    |
+| XRPL Ledger        | In progress     | No     | No          | No       | No          | No     |
```
Comment (aphelionz) [inline profile/README.md]:
```suggestion
| XRPL Ledger        | In progress     | In Progress     | No          | In Progress       | In Progress          | In Progress     |
```

File: profile/README.md [hunk 11]
```diff
@@ -1 +1,26 @@
...
+| Algorand           | No              | No     | No          | No       | No          | No     |
```
Comment (aphelionz) [inline profile/README.md]:
```suggestion
| Algorand           | In Progress              | No     | No          | No       | No          | No     |
```

File: profile/README.md [hunk 12]
```diff
@@ -1 +1,26 @@
...
+| Aleo               | No              | No     | In Progress | No       | In Progress | No     |  
+| Polkadot           | No              | No     | No          | No       | No          | No     |
+| Solana             | No              | No     | No          | No       | No          | No     |
+| Eth2 Beacon Chain  | No              | No     | No          | No       | No          | No     |
+
+
+# Contributing
+
+Feedback and Contributions from the open source community are welcome. If you have any query, you can open an issue in relevant repository and our Team will try to take it from there.
```
Comment (aphelionz) [inline profile/README.md]:
```suggestion
Feedback and Contributions from the open source community are welcome. If you have any queries, you can open an issue in the relevant repository and our team will take it from there.
```

File: profile/README.md [hunk 13]
```diff
@@ -1 +1,26 @@
-# Ziggurat
+<p align="center">
```
Comment (aphelionz) [inline profile/README.md:3]:
```suggestion
# Ziggurat 

<p align="center">
```

**Comment by aphelionz** [comment]
@muzam1l tons of little suggestions, otherwise good to go IMO

File: profile/README.md (hunk 10 above)
Comment (muzam1l) [reply profile/README.md:16]: Update Xrpl.

**Comment by muzam1l** [comment]
> ...tons of little suggestions...

Thanks, Github UI lets you easily commit the suggestions, so it was easy!

**Pull request merged**